mam/v2/examples/recv.c

```diff
@@ -135,7 +136,7 @@ int main(int ac, char **av) {
     return EXIT_FAILURE;
   }
 
-  if (mam_api_init(&api, SENDER_SEED) != RC_OK) {
+  if (mam_api_init(&api, (tryte_t *)DUMMY_SEED) != RC_OK) {
```

Comment:
Can't you just send NULL?

Reply:
A seed is needed to init the PRNG instance